app/models/search_facet.rb

```diff
@@ -12,6 +12,7 @@ class SearchFacet
   field :identifier, type: String
   field :filters, type: Array, default: []
   field :public_filters, type: Array, default: [] # filters for public studies only
+  field :merged_filters, type: Array, default: [] # filters for XDSS, includes all SCP & external facet filters
```

Comment:
```suggestion
  field :filters_with_external, type: Array, default: [] # filters for XDSS, includes all SCP & external facet filters
```
might be more explicit about what's included.  or maybe rename :filters to :scp_filters (trickier since that'll be a migration)